app/javascript/pages/admin/Reporting/index.jsx

```diff
@@ -0,0 +1,254 @@
+          <Item value="individualTag">{t('volunteer_portal.admin.tab.reporting.dropdown.nested.tag')}</Item>
+          <Item value="individualOrganization">{t('volunteer_portal.admin.tab.reporting.dropdown.nested.organization')}</Item>
+          <Item value="individualTotal">{t('volunteer_portal.admin.tab.reporting.dropdown.nested.total')}</Item>
+        </>
+      ) : (
+        <>
+          <Item value="user">{t('volunteer_portal.admin.tab.reporting.dropdown.users')}</Item>
+          <NextItem value="organized" disabled>{t('volunteer_portal.admin.tab.reporting.dropdown.organized_events')}</NextItem>
+          <NextItem value="individual" disabled>{t('volunteer_portal.admin.tab.reporting.dropdown.individual_events')}</NextItem>
+        </>
+      )}
+    </Menu>
+  )
+
+  return (
+    <AdminSection>
+      <div style={{ width: "100%" }}>
+        <ToolbarHeader>
+          <Dropdown
+              isOpen={dropDownIsOpen}
+              onSelect={handleDropdownOnSelect}
+              onStateChange={handleDropdownStateChange}
+            >
+            <Field style={{ width: 200 }}>
+              <FormLabel>{t('volunteer_portal.admin.tab.reporting.select_label')}</FormLabel>
+              <Select>
+                {
+                  report === null ? t('volunteer_portal.admin.tab.reporting.dropdown.none') : reportKeyTitleMap[report]
+                }
+              </Select>
+            </Field>
+            {renderMenu()}
+          </Dropdown>
+          <FormField>
+            <FormLabel>{t('volunteer_portal.admin.tab.reporting.filter')}</FormLabel>
+            <div style={{ marginTop: 8 }}>
+              <OfficeFilter/>
+            </div>
+          </FormField>
+          <div style={{ display: "flex" }}>
+            <FormField>
+                <FormLabel>{t('volunteer_portal.admin.tab.reporting.from')}</FormLabel>
+                <Datepicker value={dateRange.start} onChange={changeStartDate} maxValue={today} locale={i18n.language}>
+                  <CombinedInput left bare />
+                </Datepicker>
+            </FormField>
+            <FormField>
+                <FormLabel>{t('volunteer_portal.admin.tab.reporting.to')}</FormLabel>
+                <Datepicker value={dateRange.end} onChange={changeEndDate} maxValue={today} locale={i18n.language}>
+                  <CombinedInput right bare />
+                </Datepicker>
+            </FormField>
+          </div>
+          <a href={octetStream && `data:application/octet-stream;filename=export.csv,${octetStream}`} download="export.csv">
+            <Button disabled={!octetStream}>{t('volunteer_portal.admin.tab.reporting.exportascsv')}</Button>
+          </a>
+        </ToolbarHeader>
+        {
+          report === 'user' ?
+          (
+            userLoading ?
+            <div>
+              <Skeleton height="60px" />
+              {
+                Array(10).fill(0).map((value, index) => <Skeleton key={index} height="40px" />)
```

Comment:
you can use `R.range(0, 10).map((_, index) => <Skeleton key={index} height="40px" />)`